benchmarks/benchmark_throughput.py

```diff
@@ -60,8 +66,33 @@ def _get_prompt_for_image_model(question: str, *, model: str) -> str:
     raise ValueError(f"Unsupported model {model}")
 
 
+@cache
+def download_lora(lora_path: str) -> str:
+    """
+    Given an huggingface path, downloads the LoRA model and returns the path
+    on disk.
+    """
+    return snapshot_download(lora_path)
```

Comment:
We also shoudl support loading from local path .See: [get_adapter_absolute_path](https://github.com/vllm-project/vllm/blob/v0.6.5/vllm/lora/utils.py#L152)

```suggestion
    return get_adapter_absolute_path(lora_path)
```

Reply:
Updated to use `get_adapter_absolute_path` :+1: